docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,47 @@
 sidebar_label: 'What is Regression Games?'
 ---
 
-# Regression Games for Unity
+# Regression Games
 
-Regression Games is developing a platform where studios can **build and deploy bots for a variety of use cases in minutes.** We are designing for scale (deploy thousands of bots), ease of use (e.g. generative AI to build bots), ease of integration (minimal impact on game development time), and feature richness (e.g. A/B bot test for engagement).
+Regression Games is making it easy to create automated tests for your game.
 
-![vision](img/vision.png)
+## Highlighted Features
 
-## Current Stage of the Product
+| Feature | Description |
+| --- | --- |
+| [Plug-and-Play Setup](getting-started/installing-regression-games) | Simply install our SDK and add one GameObject to your scene to instantly begin using our tools. Custom integration code is rarely needed to get started. |
+| [Get Started Quickly with Recording and Input Playback](getting-started/creating-your-first-automated-test) | Developers can create functional tests by recording a play-through of their scenario and preforming validations against automated playback. |
+| [Computer Vision Capabilities](guides/interacting-with-ui-using-computer-vision) | In some cases, a bot needs to see the game to understand it. Our CV capabilities, such as OCR and object detection, can extract information from in-game screenshots to help drive behaviors and validate game state during runtime. |
+| [Exploratory Testing](guides/finding-game-breaking-bugs-with-stability-testing) | Run more than just predefined scenarios - exploratory bots such as the Monkey Bot can explore edge cases within your game. |
+| [Composability](core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments) | One approach is not always adequate for automating a certain test scenario. Our drag and drop tool allows you to stitch together and re-use bot logic across tests. |
+| [Runtime, CI/CD, and Editor Compatibility](core-concepts/running-tests-in-the-unity-test-runner) | Our tools work in the Editor, in builds, and within CI/CD. Because of our integration approach, in most cases you will **not need to restart your game** to iterate on a test scenario. |
```

Comment:
This hasn't been true for me. If the user is writing MonoBehaviours then they certainly will need to restart the game to iterate, and I anticipate that being the main use case at this stage of the product. I tried to add more detail to this statement but the stuff I tried out just reads to a user as "why are you even saying this if I don't know whether this will be true for me until I start using your SDK?"

The question after this suggestion is, what else can we write here? Because the name of the feature is now pretty much its entire description 🤔 

```suggestion
| [Runtime, CI/CD, and Editor Compatibility](core-concepts/running-tests-in-the-unity-test-runner) | Use our tools in the Editor, in builds, and within CI/CD. |
```

Reply:
The only statement that is bolded in the whole table is **not need to restart your game**, and that draws attention. Is this really the most important sentence of the whole table? I personally don't think it is that important but I might be missing some context/user feedback